modules/dynamic-plugin-proxy-service.adoc

```diff
@@ -0,0 +1,43 @@
+// Module included in the following assemblies:
+//
+// * web_console/dynamic-plugin/deploy-plugin-cluster.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="dynamic-plugin-proxy-service_{context}"]
+= Plugin service proxy
+
+You can declare a service proxy in its `ConsolePlugin` resource by using the `spec.proxy` array field if you need to make HTTP requests to an in-cluster service with your plugin. The console backend exposes the `/api/proxy/plugin/<plugin-name>/<proxy-alias>/<request-path>++?++<optional-query-parameters>` endpoint to proxy the communication between plugin and the service. A proxied request uses a _service CA bundle_ by default. The service must use HTTPS.
+
+[NOTE]
+====
+The plugin must have the `consolefetch` API to make requests from its JavaScript code or some requests might fail.
+====
+
+For each entry you must specify an endpoint and alias of the proxy under the `endpoint` and `alias` fields. For the Service proxy type, the endpoint `type` field will need to be set to `Service` and the `service` must include a `name`, `namespace`, and `port`. For example, `/api/proxy/plugin/helm/helm-charts/releases++?++limit++=++10` is a proxy request path from the `helm` plugin with a `helm-charts` service that lists ten helm releases.
+
+.Example
+[source,text,subs="+quotes,+macros"]
+----
++# ...+
+spec:
+  proxy:
+  - alias: helm-charts <1>
+    authorization: UserToken <2>
+    caCertificate: +'-----BEGIN CERTIFICATE-----\nMIID....'en+ <3>
+    endpoint: <4>
+      service:
+        name: <service-name>
+        namespace: <service-namespace>
+        port: <service-port>
+      type: Service
++# ...+
+----
+<1> Alias of the proxy.
+<2> If the service proxy request needs to contain the logged-in user's OpenShift access token, the authorization field needs to be set to `UserToken`.
+<3> If the service uses a custom service CA, the `caCertificate` field must contain the certificate bundle.
+<4> Endpoint of the proxy.
++
+[NOTE]
+====
+If the service proxy request does not contain the logged-in user's OpenShift access token, set the authorization field to `None`.
+====
```

Comment:
Since this note is about the authorization field, should it be tacked onto <2> instead of <4>?